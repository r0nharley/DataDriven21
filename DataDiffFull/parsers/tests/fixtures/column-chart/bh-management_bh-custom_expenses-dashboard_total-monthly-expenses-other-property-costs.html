<svg version="1.1" class="highcharts-root" style="font-family:&quot;Lucida Grande&quot;, &quot;Lucida Sans Unicode&quot;, Arial, Helvetica, sans-serif;font-size:12px;" xmlns="http://www.w3.org/2000/svg" width="505" height="419">
    <desc>Created with Highstock 5.0.6</desc>
    <defs>
        <clipPath id="highcharts-sk2gm76-82">
            <rect x="0" y="0" width="427" height="299" fill="none"></rect>
        </clipPath>
        <clipPath id="highcharts-sk2gm76-83">
            <rect x="0" y="0" width="427" height="299" fill="none"></rect>
        </clipPath>
    </defs>
    <rect fill="#ffffff" class="highcharts-background" x="0" y="0" width="505" height="419" rx="0" ry="0"></rect>
    <rect fill="none" class="highcharts-plot-background" x="68" y="10" width="427" height="299"></rect>
    <g class="highcharts-grid highcharts-xaxis-grid ">
        <path fill="none" class="highcharts-grid-line" d="M 103.5 10 L 103.5 309" opacity="1"></path>
        <path fill="none" class="highcharts-grid-line" d="M 138.5 10 L 138.5 309" opacity="1"></path>
        <path fill="none" class="highcharts-grid-line" d="M 174.5 10 L 174.5 309" opacity="1"></path>
        <path fill="none" class="highcharts-grid-line" d="M 209.5 10 L 209.5 309" opacity="1"></path>
        <path fill="none" class="highcharts-grid-line" d="M 245.5 10 L 245.5 309" opacity="1"></path>
        <path fill="none" class="highcharts-grid-line" d="M 281.5 10 L 281.5 309" opacity="1"></path>
        <path fill="none" class="highcharts-grid-line" d="M 316.5 10 L 316.5 309" opacity="1"></path>
        <path fill="none" class="highcharts-grid-line" d="M 352.5 10 L 352.5 309" opacity="1"></path>
        <path fill="none" class="highcharts-grid-line" d="M 387.5 10 L 387.5 309" opacity="1"></path>
        <path fill="none" class="highcharts-grid-line" d="M 423.5 10 L 423.5 309" opacity="1"></path>
        <path fill="none" class="highcharts-grid-line" d="M 458.5 10 L 458.5 309" opacity="1"></path>
        <path fill="none" class="highcharts-grid-line" d="M 494.5 10 L 494.5 309" opacity="1"></path>
        <path fill="none" class="highcharts-grid-line" d="M 67.5 10 L 67.5 309" opacity="1"></path>
    </g>
    <g class="highcharts-grid highcharts-yaxis-grid ">
        <path fill="none" stroke="#D8D8D8" stroke-width="1" stroke-dasharray="1,3" class="highcharts-grid-line" d="M 68 309.5 L 495 309.5" opacity="1"></path>
        <path fill="none" stroke="#D8D8D8" stroke-width="1" stroke-dasharray="1,3" class="highcharts-grid-line" d="M 68 266.5 L 495 266.5" opacity="1"></path>
        <path fill="none" stroke="#D8D8D8" stroke-width="1" stroke-dasharray="1,3" class="highcharts-grid-line" d="M 68 224.5 L 495 224.5" opacity="1"></path>
        <path fill="none" stroke="#D8D8D8" stroke-width="1" stroke-dasharray="1,3" class="highcharts-grid-line" d="M 68 181.5 L 495 181.5" opacity="1"></path>
        <path fill="none" stroke="#D8D8D8" stroke-width="1" stroke-dasharray="1,3" class="highcharts-grid-line" d="M 68 138.5 L 495 138.5" opacity="1"></path>
        <path fill="none" stroke="#D8D8D8" stroke-width="1" stroke-dasharray="1,3" class="highcharts-grid-line" d="M 68 95.5 L 495 95.5" opacity="1"></path>
        <path fill="none" stroke="#D8D8D8" stroke-width="1" stroke-dasharray="1,3" class="highcharts-grid-line" d="M 68 53.5 L 495 53.5" opacity="1"></path>
        <path fill="none" stroke="#D8D8D8" stroke-width="1" stroke-dasharray="1,3" class="highcharts-grid-line" d="M 68 9.5 L 495 9.5" opacity="1"></path>
    </g>
    <rect fill="none" class="highcharts-plot-border" x="68" y="10" width="427" height="299"></rect>
    <g class="highcharts-axis highcharts-xaxis ">
        <path fill="none" class="highcharts-tick" stroke="#ccd6eb" stroke-width="1" d="M 103.5 309 L 103.5 319" opacity="1"></path>
        <path fill="none" class="highcharts-tick" stroke="#ccd6eb" stroke-width="1" d="M 138.5 309 L 138.5 319" opacity="1"></path>
        <path fill="none" class="highcharts-tick" stroke="#ccd6eb" stroke-width="1" d="M 174.5 309 L 174.5 319" opacity="1"></path>
        <path fill="none" class="highcharts-tick" stroke="#ccd6eb" stroke-width="1" d="M 209.5 309 L 209.5 319" opacity="1"></path>
        <path fill="none" class="highcharts-tick" stroke="#ccd6eb" stroke-width="1" d="M 245.5 309 L 245.5 319" opacity="1"></path>
        <path fill="none" class="highcharts-tick" stroke="#ccd6eb" stroke-width="1" d="M 281.5 309 L 281.5 319" opacity="1"></path>
        <path fill="none" class="highcharts-tick" stroke="#ccd6eb" stroke-width="1" d="M 316.5 309 L 316.5 319" opacity="1"></path>
        <path fill="none" class="highcharts-tick" stroke="#ccd6eb" stroke-width="1" d="M 352.5 309 L 352.5 319" opacity="1"></path>
        <path fill="none" class="highcharts-tick" stroke="#ccd6eb" stroke-width="1" d="M 387.5 309 L 387.5 319" opacity="1"></path>
        <path fill="none" class="highcharts-tick" stroke="#ccd6eb" stroke-width="1" d="M 423.5 309 L 423.5 319" opacity="1"></path>
        <path fill="none" class="highcharts-tick" stroke="#ccd6eb" stroke-width="1" d="M 458.5 309 L 458.5 319" opacity="1"></path>
        <path fill="none" class="highcharts-tick" stroke="#ccd6eb" stroke-width="1" d="M 494.5 309 L 494.5 319" opacity="1"></path>
        <path fill="none" class="highcharts-tick" stroke="#ccd6eb" stroke-width="1" d="M 67.5 309 L 67.5 319" opacity="1"></path>
        <path fill="none" class="highcharts-axis-line" stroke="#ccd6eb" stroke-width="1" d="M 68 309.5 L 495 309.5"></path>
    </g>
    <g class="highcharts-axis highcharts-yaxis ">
        <path fill="none" class="highcharts-axis-line" d="M 68 10 L 68 309"></path>
    </g>
    <g class="highcharts-series-group">
        <g class="highcharts-series highcharts-series-0 highcharts-column-series highcharts-color-undefined highcharts-tracker " transform="translate(68,10) scale(1 1)" width="299" height="427" clip-path="url(http://local.dev/bh-management/bi/bh-custom-expenses-dashboard/visualization#highcharts-sk2gm76-83)">
            <rect x="8.5" y="242.5" width="18" height="57" fill="#6eda55" class="highcharts-point" stroke="#ffffff" stroke-width="1"></rect>
            <rect x="44.5" y="225.5" width="18" height="74" fill="#ff8c88" class="highcharts-point" stroke="#ffffff" stroke-width="1"></rect>
            <rect x="79.5" y="223.5" width="18" height="76" fill="#ff8c88" class="highcharts-point" stroke="#ffffff" stroke-width="1"></rect>
            <rect x="115.5" y="194.5" width="18" height="105" fill="#6eda55" class="highcharts-point" stroke="#ffffff" stroke-width="1"></rect>
            <rect x="151.5" y="154.5" width="18" height="145" fill="#ffd801" class="highcharts-point" stroke="#ffffff" stroke-width="1"></rect>
            <rect x="186.5" y="68.5" width="18" height="231" fill="#ff8c88" class="highcharts-point" stroke="#ffffff" stroke-width="1"></rect>
            <rect x="222.5" y="49.5" width="18" height="250" fill="#6eda55" class="highcharts-point" stroke="#ffffff" stroke-width="1"></rect>
            <rect x="257.5" y="81.5" width="18" height="218" fill="#6eda55" class="highcharts-point " stroke="#ffffff" stroke-width="1"></rect>
            <rect x="293.5" y="74.5" width="18" height="225" fill="#6eda55" class="highcharts-point" stroke="#ffffff" stroke-width="1"></rect>
            <rect x="329.5" y="59.5" width="18" height="240" fill="#ff8c88" class="highcharts-point" stroke="#ffffff" stroke-width="1"></rect>
            <rect x="364.5" y="80.5" width="18" height="219" fill="#6eda55" class="highcharts-point" stroke="#ffffff" stroke-width="1"></rect>
            <rect x="400.5" y="56.5" width="18" height="243" fill="#6eda55" class="highcharts-point" stroke="#ffffff" stroke-width="1"></rect>
        </g>
        <g class="highcharts-markers highcharts-series-0 highcharts-column-series highcharts-color-undefined " transform="translate(68,10) scale(1 1)" width="299" height="427" clip-path="none"></g>
        <g class="highcharts-series highcharts-series-1 highcharts-spline-series highcharts-color-undefined " transform="translate(68,10) scale(1 1)" width="299" height="427" clip-path="url(http://local.dev/bh-management/bi/bh-custom-expenses-dashboard/visualization#highcharts-sk2gm76-83)">
            <path fill="none" d="M 17.791666666667 235.37748148571433 C 17.791666666667 235.37748148571433 39.1416666666668 244.85426085714286 53.375 244.85426085714286 C 67.6083333333332 244.85426085714286 74.7249999999998 244.85426085714286 88.958333333333 238.32877807142862 C 103.19166666666779 231.80329528571437 110.30833333333521 197.95746831714447 124.54166666667 181.30974717142857 C 138.775000000002 164.6620260257159 145.891666666668 168.34821079142864 160.125 155.0901723428572 C 174.35833333333198 141.83213389428576 181.474999999998 139.74617149999648 195.70833333333 115.01955492857138 C 209.941666666666 90.29293835713928 217.058333333334 53.268289471431615 231.29166666667 31.45708948571422 C 245.525000000002 9.645889500003008 252.64166666666802 5.963554999999872 266.875 5.963554999999872 C 281.10833333333204 5.963554999999872 288.224999999998 13.624528728573566 302.45833333333 29.87976197142848 C 316.691666666666 46.134995214287876 323.808333333334 87.23972121428565 338.04166666667 87.23972121428565 C 352.275000000002 87.23972121428565 359.39166666666796 80.80070581142853 373.625 73.59402328571426 C 387.85833333333204 66.38734075999999 409.20833333333 51.20630858571428 409.20833333333 51.20630858571428" class="highcharts-graph" stroke="#808080" stroke-width="2" stroke-linejoin="round" stroke-linecap="round"></path>
            <path fill="none" d="M 7.791666666666998 235.37748148571433 L 17.791666666667 235.37748148571433 C 17.791666666667 235.37748148571433 39.1416666666668 244.85426085714286 53.375 244.85426085714286 C 67.6083333333332 244.85426085714286 74.7249999999998 244.85426085714286 88.958333333333 238.32877807142862 C 103.19166666666779 231.80329528571437 110.30833333333521 197.95746831714447 124.54166666667 181.30974717142857 C 138.775000000002 164.6620260257159 145.891666666668 168.34821079142864 160.125 155.0901723428572 C 174.35833333333198 141.83213389428576 181.474999999998 139.74617149999648 195.70833333333 115.01955492857138 C 209.941666666666 90.29293835713928 217.058333333334 53.268289471431615 231.29166666667 31.45708948571422 C 245.525000000002 9.645889500003008 252.64166666666802 5.963554999999872 266.875 5.963554999999872 C 281.10833333333204 5.963554999999872 288.224999999998 13.624528728573566 302.45833333333 29.87976197142848 C 316.691666666666 46.134995214287876 323.808333333334 87.23972121428565 338.04166666667 87.23972121428565 C 352.275000000002 87.23972121428565 359.39166666666796 80.80070581142853 373.625 73.59402328571426 C 387.85833333333204 66.38734075999999 409.20833333333 51.20630858571428 409.20833333333 51.20630858571428 L 419.20833333333 51.20630858571428" stroke-linejoin="round" visibility="visible" stroke="rgba(192,192,192,0.0001)" stroke-width="22" class="highcharts-tracker"></path>
        </g>
        <g class="highcharts-markers highcharts-series-1 highcharts-spline-series highcharts-color-undefined highcharts-tracker" transform="translate(68,10) scale(1 1)" width="299" height="427" clip-path="none"></g>
    </g>
    <g class="highcharts-legend" transform="translate(136,378)">
        <rect fill="none" class="highcharts-legend-box" rx="0" ry="0" x="0" y="0" width="232" height="26" visibility="visible"></rect>
        <g>
            <g>
                <g class="highcharts-legend-item highcharts-column-series highcharts-color-undefined highcharts-series-0" transform="translate(8,3)">
                    <text x="21" style="color:#333333;font-size:12px;font-weight:normal;cursor:pointer;fill:#333333;" text-anchor="start" y="15">
                        <tspan>Total Actual</tspan>
                    </text>
                    <rect x="2" y="4" width="12" height="12" fill="#D3D3D3" rx="6" ry="6" class="highcharts-point"></rect>
                </g>
                <g class="highcharts-legend-item highcharts-spline-series highcharts-color-undefined highcharts-series-1" transform="translate(124.296875,3)">
                    <path fill="none" d="M 0 11 L 16 11" class="highcharts-graph" stroke="#808080" stroke-width="2"></path>
                    <text x="21" y="15" style="color:#333333;font-size:12px;font-weight:normal;cursor:pointer;fill:#333333;" text-anchor="start">
                        <tspan>Total Budget</tspan>
                    </text>
                </g>
            </g>
        </g>
    </g>
    <g class="highcharts-axis-labels highcharts-xaxis-labels ">
        <text x="88.38439153101734" style="color:#666666;cursor:default;font-size:11px;fill:#666666;width:419px;text-overflow:ellipsis;" text-anchor="end" transform="translate(0,0) rotate(-45 88.38439153101734 325)" y="325" opacity="1">
            <tspan>Jul 2016</tspan>
        </text>
        <text x="123.9677248643507" style="color:#666666;cursor:default;font-size:11px;fill:#666666;width:419px;text-overflow:ellipsis;" text-anchor="end" transform="translate(0,0) rotate(-45 123.9677248643507 325)" y="325" opacity="1">
            <tspan>Aug 2016</tspan>
        </text>
        <text x="159.55105819768403" style="color:#666666;cursor:default;font-size:11px;fill:#666666;width:419px;text-overflow:ellipsis;" text-anchor="end" transform="translate(0,0) rotate(-45 159.55105819768403 325)" y="325" opacity="1">
            <tspan>Sep 2016</tspan>
        </text>
        <text x="195.13439153101737" style="color:#666666;cursor:default;font-size:11px;fill:#666666;width:419px;text-overflow:ellipsis;" text-anchor="end" transform="translate(0,0) rotate(-45 195.13439153101737 325)" y="325" opacity="1">
            <tspan>Oct 2016</tspan>
        </text>
        <text x="230.71772486435069" style="color:#666666;cursor:default;font-size:11px;fill:#666666;width:419px;text-overflow:ellipsis;" text-anchor="end" transform="translate(0,0) rotate(-45 230.71772486435069 325)" y="325" opacity="1">
            <tspan>Nov 2016</tspan>
        </text>
        <text x="266.301058197684" style="color:#666666;cursor:default;font-size:11px;fill:#666666;width:419px;text-overflow:ellipsis;" text-anchor="end" transform="translate(0,0) rotate(-45 266.301058197684 325)" y="325" opacity="1">
            <tspan>Dec 2016</tspan>
        </text>
        <text x="301.88439153101734" style="color:#666666;cursor:default;font-size:11px;fill:#666666;width:419px;text-overflow:ellipsis;" text-anchor="end" transform="translate(0,0) rotate(-45 301.88439153101734 325)" y="325" opacity="1">
            <tspan>Jan 2017</tspan>
        </text>
        <text x="337.46772486435066" style="color:#666666;cursor:default;font-size:11px;fill:#666666;width:419px;text-overflow:ellipsis;" text-anchor="end" transform="translate(0,0) rotate(-45 337.46772486435066 325)" y="325" opacity="1">
            <tspan>Feb 2017</tspan>
        </text>
        <text x="373.05105819768403" style="color:#666666;cursor:default;font-size:11px;fill:#666666;width:419px;text-overflow:ellipsis;" text-anchor="end" transform="translate(0,0) rotate(-45 373.05105819768403 325)" y="325" opacity="1">
            <tspan>Mar 2017</tspan>
        </text>
        <text x="408.63439153101734" style="color:#666666;cursor:default;font-size:11px;fill:#666666;width:419px;text-overflow:ellipsis;" text-anchor="end" transform="translate(0,0) rotate(-45 408.63439153101734 325)" y="325" opacity="1">
            <tspan>Apr 2017</tspan>
        </text>
        <text x="444.21772486435066" style="color:#666666;cursor:default;font-size:11px;fill:#666666;width:419px;text-overflow:ellipsis;" text-anchor="end" transform="translate(0,0) rotate(-45 444.21772486435066 325)" y="325" opacity="1">
            <tspan>May 2017</tspan>
        </text>
        <text x="479.80105819768403" style="color:#666666;cursor:default;font-size:11px;fill:#666666;width:419px;text-overflow:ellipsis;" text-anchor="end" transform="translate(0,0) rotate(-45 479.80105819768403 325)" y="325" opacity="1">
            <tspan>Jun 2017</tspan>
        </text>
    </g>
    <g class="highcharts-axis-labels highcharts-yaxis-labels ">
        <text x="53" style="color:#666666;cursor:default;font-size:11px;fill:#666666;width:157px;text-overflow:clip;" text-anchor="end" transform="translate(0,0)" y="314" opacity="1">
            <tspan>$600K</tspan>
        </text>
        <text x="53" style="color:#666666;cursor:default;font-size:11px;fill:#666666;width:157px;text-overflow:clip;" text-anchor="end" transform="translate(0,0)" y="271" opacity="1">
            <tspan>$700K</tspan>
        </text>
        <text x="53" style="color:#666666;cursor:default;font-size:11px;fill:#666666;width:157px;text-overflow:clip;" text-anchor="end" transform="translate(0,0)" y="228" opacity="1">
            <tspan>$800K</tspan>
        </text>
        <text x="53" style="color:#666666;cursor:default;font-size:11px;fill:#666666;width:157px;text-overflow:clip;" text-anchor="end" transform="translate(0,0)" y="185" opacity="1">
            <tspan>$900K</tspan>
        </text>
        <text x="53" style="color:#666666;cursor:default;font-size:11px;fill:#666666;width:157px;text-overflow:clip;" text-anchor="end" transform="translate(0,0)" y="143" opacity="1">
            <tspan>$1M</tspan>
        </text>
        <text x="53" style="color:#666666;cursor:default;font-size:11px;fill:#666666;width:157px;text-overflow:clip;" text-anchor="end" transform="translate(0,0)" y="100" opacity="1">
            <tspan>$1.10M</tspan>
        </text>
        <text x="53" style="color:#666666;cursor:default;font-size:11px;fill:#666666;width:157px;text-overflow:clip;" text-anchor="end" transform="translate(0,0)" y="57" opacity="1">
            <tspan>$1.20M</tspan>
        </text>
        <text x="53" style="color:#666666;cursor:default;font-size:11px;fill:#666666;width:157px;text-overflow:clip;" text-anchor="end" transform="translate(0,0)" y="15" opacity="1">
            <tspan>$1.30M</tspan>
        </text>
    </g>
    <rect x="205" y="10" width="47" height="299" fill="rgba(51,92,173,0.25)" class="highcharts-selection-marker"></rect>
    <g class="highcharts-label highcharts-tooltip highcharts-color-undefined" style="cursor:default;pointer-events:none;white-space:nowrap;" transform="translate(280,-9999)" opacity="0" visibility="visible">
        <path fill="rgba(255, 255, 255, 0)" class="highcharts-label-box highcharts-tooltip-box" d="M 0 0 L 49 0 55 -6 61 0 111 0 C 111 0 111 0 111 0 L 111 102 C 111 102 111 102 111 102 L 0 102 C 0 102 0 102 0 102 L 0 0 C 0 0 0 0 0 0"></path>
    </g>
</svg>