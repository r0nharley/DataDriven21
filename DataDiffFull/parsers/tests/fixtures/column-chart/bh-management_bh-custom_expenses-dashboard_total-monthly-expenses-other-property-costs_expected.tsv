Jul 2016	Total Actual	732274.247492	#6eda55
Jul 2016	Total Budget	748949.04
Aug 2016	Total Actual	772073.578595	#ff8c88
Aug 2016	Total Budget	726762.6
Sep 2016	Total Actual	776755.852843	#ff8c88
Sep 2016	Total Budget	742039.65
Oct 2016	Total Actual	844648.829431	#6eda55
Oct 2016	Total Budget	875529.02
Nov 2016	Total Actual	938294.314381	#ffd801
Nov 2016	Total Budget	936912.64
Dec 2016	Total Actual	1139632.10702	#ff8c88
Dec 2016	Total Budget	1030723.45
Jan 2017	Total Actual	1184113.71237	#6eda55
Jan 2017	Total Budget	1226354.64
Feb 2017	Total Actual	1109197.32441	#6eda55
Feb 2017	Total Budget	1286038.5
Mar 2017	Total Actual	1125585.28428	#6eda55
Mar 2017	Total Budget	1230047.38
Apr 2017	Total Actual	1160702.34114	#ff8c88
Apr 2017	Total Budget	1095759.85
May 2017	Total Actual	1111538.46154	#6eda55
May 2017	Total Budget	1127706.3
Jun 2017	Total Actual	1167725.75251	#6eda55
Jun 2017	Total Budget	1180119.01